.github/workflows/rust.yml

```
@@ -110,6 +110,24 @@ jobs:
         run: |
           ./android-build cargo build --target="armv7-linux-androideabi"
 
+  cross:
+    name: Cross ${{ matrix.target }}
```

Just for the sake of clarity and consistency with the other job names:

```suggestion
  linux-cross-compile:
    name: linux (${{ matrix.target }})
```